pkg/services/images.go

```diff
@@ -27,6 +28,7 @@ var WaitGroup sync.WaitGroup
 // the business logic of creating RHEL For Edge Images
 type ImageServiceInterface interface {
 	CreateImage(image *models.Image, account string) error
+	UpdateImage(image *models.Image, account string, previous_image *models.Image) error
```

Comment:
```suggestion
	UpdateImage(image *models.Image, account string, previousImage *models.Image) error
```

just being strangely attached to Go's camelCase suggestion here 😺